tests/unit/hive_metastore/test_principal_grants.py

```diff
@@ -482,10 +482,11 @@ def test_get_eligible_locations_principals_aws(ws, installation):
     )
     locations = aws_acl(ws, installation)
     eligible_locations = locations.get_eligible_locations_principals()
-    assert len(eligible_locations) == 3
-    assert eligible_locations[0].locations == {'s3://storage5/folder5': 'WRITE_FILES'}
-    assert eligible_locations[1].locations == {'s3://storage3/folder3': 'WRITE_FILES'}
-    assert eligible_locations[1].locations == {'s3://storage3/folder3': 'WRITE_FILES'}
```

Comment:
This line is a duplicate of the line above, and not checking the third location as intended. (That's what triggered me to make this PR.)